rust/relay/server/src/main.rs

```diff
@@ -152,7 +169,7 @@ async fn try_main(args: Args) -> Result<()> {
     let mut ebpf = args
         .ebpf_offloading
         .as_deref()
-        .map(|interface| ebpf::Program::try_load(interface, &args.ebpf_attach_mode))
+        .map(|interface| ebpf::Program::try_load(interface, args.ebpf_attach_mode.as_str()))
```

Comment:
Why pass it as str here when you are only reparsing it again after to map to the other mode?

That roundtrip seems unnecessary.